AppKit/CPWindow/CPWindow.j

```diff
@@ -3844,6 +3814,263 @@ var interpolate = function(fromValue, toValue, progress)
+                continue;
+            
+            if ((trackingOptions & CPTrackingActiveAlways) ||
+                (trackingOptions & CPTrackingActiveInActiveApp) ||
+                ((trackingOptions & CPTrackingActiveInKeyWindow) && isKeyWindow) ||
+                ((trackingOptions & CPTrackingActiveWhenFirstResponder) && isKeyWindow && (_firstResponder === aView)))
+            {
+                if (!(CGRectContainsPoint([aTrackingArea actualRect], point)))
+                    continue;
+                
+                [mouseEnteredStack addObject:aTrackingArea];
+                
+                if ([_mouseEnteredStack containsObject:aTrackingArea])
+                {
+                    // Mouse was already in this rect so it's a mouseMoved
+                    
+                    if (!dragging && (trackingOptions & CPTrackingMouseMoved))
+                        [[aTrackingArea owner] mouseMoved:anEvent];
+                }
+                else
+                {
+                    // Mouse was not in this rect so it's a mouseEntered
+                    
+                    if (trackingOptions & CPTrackingMouseEnteredAndExited)
+                        [[aTrackingArea owner] mouseEntered:[CPEvent enterExitEventWithType:CPMouseEntered
+                                                                                   location:point
+                                                                              modifierFlags:[anEvent modifierFlags]
+                                                                                  timestamp:[anEvent timestamp]
+                                                                               windowNumber:_windowNumber
+                                                                                    context:nil
+                                                                                eventNumber:-1
+                                                                               trackingArea:aTrackingArea]];
+                }
+                
+                if (trackingOptions & CPTrackingCursorUpdate)
+                    [cursorUpdateStack addObject:aTrackingArea];
+            }
+        }
+    }
+    
+    // Search now for exited views (were in _mouseEnteredStack but no more in mouseEnteredStack)
+    
+    for (var i = 0; i < _mouseEnteredStack.length; i++)
+    {
+        var aTrackingArea   = _mouseEnteredStack[i],
+            trackingOptions = [aTrackingArea options];
+        
+        if (![mouseEnteredStack containsObject:aTrackingArea])
+        {
+            // Mouse is no more in this area so it's a mouseExited
+            
+            if ((trackingOptions & CPTrackingMouseEnteredAndExited) && (!dragging || (dragging && (trackingOptions & CPTrackingEnabledDuringMouseDrag))))
+                [[aTrackingArea owner] mouseExited:[CPEvent enterExitEventWithType:CPMouseExited
```

Comment:
You have to check if the owner implements mouseExited:, it may not be a CPResponder subclass.